src/Vec3.zig

```diff
@@ -23,6 +23,26 @@ pub const Vec3 = struct {
         };
     }
 
+    pub fn mul(self: *const Self, other: Self) Vec3 {
```

Comment:
Please call it `mulVec3` in case we need some `f32` multiplications later.

Reply:
done